spring-kafka-docs/src/main/asciidoc/testing.adoc

```diff
@@ -214,6 +215,24 @@ IMPORTANT: The preceding example provides no mechanism for shutting down the bro
 This could be a problem if, say, you run your tests in a Gradle daemon.
 You should not use this technique in such a situation, or you should use something to call `destroy()` on the `EmbeddedKafkaBroker` when your tests are complete.
 
+Starting with version 3.0, the framework exposes a `GlobalEmbeddedKafkaTestExecutionListener` for JUnit Jupiter Platform, which is disabled by default.
+The purpose of this listener is to start one global `EmbeddedKafkaBroker` for the whole test plan and stop it in the end of plan.
+To enable this listener, and therefore have a single global embedded Kafka cluster for all the tests in the project, a `spring.kafka.global.embedded.enabled` property must be set to `true` via system properties or JUnit platform configuration.
```

Comment:
```suggestion
Starting with version 3.0, the framework exposes a `GlobalEmbeddedKafkaTestExecutionListener` for the JUnit Jupiter Platform; it is disabled by default.
The purpose of this listener is to start one global `EmbeddedKafkaBroker` for the whole test plan and stop it at the end of the plan.
To enable this listener, and therefore have a single global embedded Kafka cluster for all the tests in the project, the `spring.kafka.global.embedded.enabled` property must be set to `true` via system properties or JUnit platform configuration.
```

Reply:
It's actually for the "JUnit Platform" (a.k.a., JUnit 5).

It's not specific to JUnit Jupiter.